README.md

```diff
@@ -210,106 +185,91 @@ Further, the key manager interface **must** be passed as an argument to _all_ pu
-| Verifiable Presentations as `vp-jwt` | ⚠️          | ⚠️      | ❌   | ❌    |
-
-### [W3C Verifiable Credential Data Model 2.0](https://www.w3.org/TR/vc-data-model-2.0/)
-
-| Feature                   | Typescript | Kotlin | Rust | Swift |
-| ------------------------- | ---------- | ------ | ---- | ----- |
-| Creation                  | ❌         | ❌     | ❌   | ❌    |
-| Signing as `vc-jose-cose` | ❌         | ❌     | ❌   | ❌    |
-| Verification              | ❌         | ❌     | ❌   | ❌    |
-| Validation                | ❌         | ❌     | ❌   | ❌    |
-
-### [SD-JWT](https://datatracker.ietf.org/doc/draft-ietf-oauth-selective-disclosure-jwt/) / [SD-JWT-VC](https://datatracker.ietf.org/doc/draft-ietf-oauth-sd-jwt-vc/)
-
-| Feature             | Typescript | Kotlin | Rust | Swift |
-| ------------------- | ---------- | ------ | ---- | ----- |
-| Creation            | ❌         | ❌     | ❌   | ❌    |
-| Signing             | ❌         | ❌     | ❌   | ❌    |
-| Verification        | ❌         | ❌     | ❌   | ❌    |
-| Validation          | ❌         | ❌     | ❌   | ❌    |
-
-### [Bitstring Status List](https://w3c.github.io/vc-bitstring-status-list/)
-
-| Feature                           | Typescript | Kotlin | Rust | Swift |
-| --------------------------------- | ---------- | ------ | ---- | ----- |
-| Creation using `vc-jwt`           | ❌         | ❌     | ❌   | ❌   |
-| Validation using `vc-jwt`         | ❌         | ❌     | ❌   | ❌   |
-
-### [VC JSON Schema](https://www.w3.org/TR/vc-json-schema/)
-
-| Feature                           | Typescript | Kotlin | Rust | Swift |
-| --------------------------------- | ---------- | ------ | ---- | ----- |
-| Creation `JsonSchema`             | ❌         | ❌     | ❌   | ❌   |
-| Creation `JsonSchemaCredential`   | ❌         | ❌     | ❌   | ❌   |
-| Validation `JsonSchema`           | ❌         | ❌     | ❌   | ❌   |
-| Validation `JsonSchemaCredential` | ❌         | ❌     | ❌   | ❌   |
-
-### [Presentation Exchange V2](https://identity.foundation/presentation-exchange/spec/v2.0.0/)
-
-| Feature                | Typescript | Kotlin | Rust | Swift |
-| ---------------------- | ---------- | ------ | ---- | ----- |
-| Concrete Type          | ✅         | ✅     | ❌   | ❌    |
-| Validation             | ✅         | ✅     | ❌   | ❌    |
-| Credential Evaluation  | ✅         | ✅     | ❌   | ❌    |
-| [Predicates](https://identity.foundation/presentation-exchange/spec/v2.0.0/#predicate-feature)                           | ✅         | ✅     | ❌   | ❌    |
-| [Relational Constraints](https://identity.foundation/presentation-exchange/spec/v2.0.0/#relational-constraint-feature)   | ✅         | ❌     | ❌   | ❌    |
-| [Credential Status](https://identity.foundation/presentation-exchange/spec/v2.0.0/#credential-status-constraint-feature) | ❌         | ❌     | ❌   | ❌    |
+| Feature                  |
+| ------------------------ |
+| Data model               | 
+| Signing and verification | 
```

Comment:
No `Validation` feature here?

Reply:
+1 to validation ?
Also, are we going to implement the "decoys" functionality for sd-jwt?

Reply:
I will add validation back